tools/tune/dmic/dmic_fir.m

```diff
@@ -46,7 +46,7 @@
 
 %% Iterate order and passband width
 iterate = 1;
-lo_ind = find(f < passhz/20);
+lo_ind = f < passhz/20;
```

Comment:
I like find() in this know it's not boolean comparison. Does this cause issues?

Reply:
The change is proposed support `h_lo = freqz(b, 1, f(lo_ind), fs_fir);` where `f` is an array, The performance is enhanced by using logical indexing rather than `find`